### PR TITLE
chore: preformat cloudrad docs

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**
Fixes: 
* https://github.com/googleapis/google-cloud-ruby/issues/13167
* https://github.com/googleapis/google-cloud-ruby/issues/13168
* https://github.com/googleapis/google-cloud-ruby/issues/13171
* https://github.com/googleapis/google-cloud-ruby/issues/13172
* https://github.com/googleapis/google-cloud-ruby/issues/13269

### CODE REVIEW

File: google-cloud-vision-v1/Rakefile [hunk 1]
```diff
@@ -161,7 +161,7 @@ end
 
 task :cloudrad do
   ENV["CLOUDRAD_GEM_NAME"] = "google-cloud-vision-v1"
-  sh "bundle exec yard doc --fail-on-warning --yardopts .yardopts-cloudrad"
+  sh "bundle exec yard doc --fail-on-warning --yardopts .yardopts-cloudrad --no-progress"
```
Comment (dazuma) [inline google-cloud-vision-v1/Rakefile:164]: This is going to get overwritten by the generator. Do we need to update the generator to do this for all clients?

File: Rakefile [hunk 2]
```diff
@@ -508,8 +508,8 @@ namespace :kokoro do
   end
 
   task :publish_docs do
-    # kokoro.devsite
-    kokoro.cloudrad
+    kokoro.devsite
+    # kokoro.cloudrad
```
Comment (dazuma) [inline Rakefile:512]: This code from the rakefile and rakelib is actually dead now. The logic was moved into `.toys/kokoro/.lib/releaser.rb`.

File: rakelib/kokoro/kokoro.rb [hunk 3]
```diff
@@ -163,6 +163,32 @@ def all_local_docs_tests only_updated: false
     end
   end
 
+  def one_local_cloudrad_docs gem
```
Comment (dazuma) [inline rakelib/kokoro/kokoro.rb:166]: This class is (mostly?) dead. The logic was moved to `.toys/kokoro/.lib/releaser.rb` a few months ago.

File: google-cloud-vision-v1/Rakefile (hunk 1 above)
Comment (TheRoyalTnetennba) [reply google-cloud-vision-v1/Rakefile:164]: woops that was a debug thing. removing

**Review by dazuma — APPROVED** [review]

**Pull request merged**